core/payment/examples/invoice_flow.rs

```diff
@@ -7,9 +7,14 @@ use ya_client_model::payment::{Acceptance, DocumentStatus, NewAllocation, NewInv
 
 #[actix_rt::main]
 async fn main() -> anyhow::Result<()> {
+    std::env::set_var("RUST_LOG", "info");
```

Comment:
but what if i want to run the example with debug ( same as previous comment )

Reply:
Fixed.